### PR TITLE
Add MoreTweaker to the documentation

### PR DESCRIPTION
**Pull request by noeppi-noeppi**

### CODE REVIEW

File: docs/Mods/MoreTweaker/Railcraft/CokeOven.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```zenscript
+//moretweaker.railcraft.CokeOven.add(IItemStack output, IIngredient input, ILiquidStack liquidOutput, optional int ticks);
+moretweaker.railcraft.CokeOven.add(<minecraft:leather>, <minecraft:rotten_flesh>, <liquid:water> * 10, 300);
+```
+
+## Removal
+
+```zenscript
+//moretweaker.railcraft.CokeOven.remove(IIngredient output);
+moretweaker.railcraft.CokeOven.remove(<ore:fuelCoke>);
```
Comment (kindlich) [inline docs/Mods/MoreTweaker/Railcraft/CokeOven.md:17]: [Technical Comment]
Same as above already:
ILiquidStack extends IIngredient, so people may think they can remove by liquid output as well.

Please either add a comment that this doesn't work or make it work :stuck_out_tongue:

File: docs/Mods/MoreTweaker/Railcraft/RockCrusher.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Rock Crusher
+
+## Package
+`moretweaker.railcraft.RockCrusher`
+
+## Addition
+
+```zenscript
+//moretweaker.railcraft.RockCrusher.add(IIngredient input, IItemStack[] outputs, float[] chances, optional int ticks);
```
Comment (kindlich) [inline docs/Mods/MoreTweaker/Railcraft/RockCrusher.md:9]: [Technical Question]
Why not use IWeightedItemStack[] here?
Then you could do
`[<minecraft:coal> % 95, <minecraft:dirt>.weight(0.25)]`
And CrT would auto-convert when no chance was given to 100%

File: docs/Mods/MoreTweaker/Railcraft/RollingMachine.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Rolling Machine
+
+## Package
+`moretweaker.railcraft.RollingMachine`
+
+## Addition
+
```
Comment (kindlich) [inline docs/Mods/MoreTweaker/Railcraft/RollingMachine.md:7]: [Technical Question]
Why not directly create a CrT Shaped/Shapeless recipe from the given inputs/output and give that to the Crafter?
The classes are MCRecipeShaped/Shapeless iirc

File: docs/Mods/MoreTweaker/VoidCraft/VoidAlchemy.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Voidic Alchemy Table
+
+## Package
+`moretweaker.voidcraft.VoidAlchemy`
+
+## Addition
+
+```zenscript
+//moretweaker.voidcraft.VoidAlchemy.add(IItemStack output, IIngredient[] inputs, optional int voidicPower);
```
Comment (kindlich) [inline docs/Mods/MoreTweaker/VoidCraft/VoidAlchemy.md:9]: [Technical Question]
Why not use 6 parameters with defaults?

**Comment by noeppi-noeppi** [comment]
Yes, I am the author of MoreTweaker. Thank you for your feedback. 


*You didn't add the Erebus support here, as well as AoA and Galacticraft, is that intended?*

So it is intended that AoA is not in the documentation as I could not test it and I don't know if it works (See [here](https://www.curseforge.com/minecraft/mc-mods/moretweaker#c33)). Erebus and Galacticraft should be here. I think I forgot them.


*you can in many occasions use the CrafttweakerMC methods to deal with CrT's types to keep your logic simpler.*

That is true, I am going to do that.


*When you use IIngredient, you only use its items functionality, not things like Transformers, correct?*

Yes, I am using `IIngredient` as it's a superinterface of `IItemStack` and `IOredictEntry`. I might change this in the future.


*For your IAction#describe: ...*

That is true as well. I'll change it.


*Why do you generally add multiple recipes for a given input?*

Some mods do not support the ore dictionary in their recipe system and I still want to write <ore:ingotCopper> instead of <somemod:copper_ingot> and <anothermod:ingot_copper> and <thirdmod:copper>. It's much simpler to just write one statement with <ore:ingotCopper>.

Seems like I have some work for the next couple of days. I am going to implement the changes requested by you (first in the mod, then here in my PR for the documentation.). I'll mark this PR as a draft until I am done.

(I did not even know that CraftTweaker could do all this... I was using it a lot in 1.7.10 based on [this](http://minetweaker3.powerofbytes.com/wiki/Main_Page#Tutorials_.281.7.X.29) guide. Then I have not used for a longer period of time...)

**Pull request closed without merging**